maplayer.c

```diff
@@ -1604,9 +1604,19 @@ char* msLayerBuildSQLOrderBy(layerObj *layer)
   if( layer->sortBy.nProperties > 0 ) {
     int i;
     for(i=0;i<layer->sortBy.nProperties;i++) {
-      char* escaped = msSmallMalloc(strlen(layer->sortBy.properties[i].item) + 3);
-      sprintf(escaped, "\"%s\"", layer->sortBy.properties[i].item);
-      if( i > 0 )
+      char* escaped = msLayerEscapePropertyName(layer, layer->sortBy.properties[i].item);
+      //Enclose property name in double quotes (if it isn't yet) to ensure that mixed case property names are supported
+      if (escaped[0] != '"')
```

Comment:
`msOracleSpatialEscapePropertyName` is not enclosing a propertyname with double quotes - it just duplicates enclosing double quotes and backslashes, so e.g. it modifies `"PropertyName"` into `""PropertyName""`. Don't ask me if this really makes sense...
So could you imagine that we just modify `msOracleSpatialEscapePropertyName` to do what you think it should do? e.g. modifies `PropertyName` into `"PropertyName"`?